docker/Dockerfile

```diff
@@ -1,35 +1,22 @@
-# syntax=docker/dockerfile:1
-FROM ubuntu:22.04
+# AMD64 needed explicitly on ARM as some sofwtare is only available in AMD64
+FROM --platform=amd64 mambaorg/micromamba:1.5.1-jammy
```

Comment:
Can't we go to a 2.x version of micromamba?

Reply:
Yup, done